### PR TITLE
Next frontend

### PR DESCRIPTION
**Pull request by imxeno**
Resolves L2B-5506, L2B-5507, L2B-5568.

### CODE REVIEW

File: packages/frontend2/next.config.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { withPlausibleProxy } from 'next-plausible'
+import { env } from './src/env.js'
+
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  // biome-ignore lint/suspicious/useAwait: rewrites must be
```
Comment (torztomasz) [inline packages/frontend2/next.config.js:6]: must be what?

File: packages/frontend2/src/app/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}>) {
+  return (
+    <html lang="en">
+      <body className={roboto.className}>
+        <PlausibleProvider
+          domain={env.NEXT_PUBLIC_PLAUSIBLE_DOMAIN}
+          enabled={env.NEXT_PUBLIC_PLAUSIBLE_ENABLED}
+        >
+          <TRPCReactProvider>{children}</TRPCReactProvider>
+        </PlausibleProvider>
```
Comment (torztomasz) [inline packages/frontend2/src/app/layout.tsx:38]: not sure if its correct moment, but we should create some `withProviders` stuff

**Review by torztomasz — APPROVED** [review]

File: packages/frontend2/src/app/layout.tsx (hunk 2 above)
Comment (imxeno) [reply packages/frontend2/src/app/layout.tsx:38]: I guess there won't be that many of them, so I guess it's not really needed...? Anyway we can add it later if we feel like it's worth.

File: packages/frontend2/next.config.js (hunk 1 above)
Comment (imxeno) [reply packages/frontend2/next.config.js]: async, fixed!

**Pull request merged**